### ISSUE TITLE
Issue after onDocumentLoad!

### ISSUE THREAD

**Issue by nish5uec** [react-pdf-viewer/react-pdf-viewer#556]
I have a use case where need to show multiple pdf documents with option to navigate through them. While the first document load fine with highlight and jump functionality, i use onDocumentLoad method once user navigate to next document to highlight selected keyword. After this, highlight functionality works with newly loaded functionality but jump functionality seems to be broken. Seems like issue with state management of jumpstate. Can you please take a look and let me know.

Basically the use case is to select a keyword to highlight and navigate through multiple documents with keyword highlighted. The jump functionality works fine with first document but breaks from second document. 

As mentioned, i am using onDocumentLoad to highlight the keyword as user navigate to second document, is there any other way to accomplish this or is it a bug?

**Comment by phuoc-ng** [comment]
@nish5uec 
Please take a look at this example https://codesandbox.io/s/react-pdf-viewer-jrxh8
